### DISCUSSION TITLE
How do I create tests in '*.spec.ts' files instead of '*.test.ts' with Copilot Chat?

### DISCUSSION THREAD

**Discussion by smallstepstoday** [community/community#56329] [Copilot Conversations]
### Select Topic Area

Question

### Body

The ability to append a new test into a test file is super helpful. However, I do not see a way to configure Copilot to add a test into the associated `spec` file. When I ask it to create a test for a function in `some-file.ts`, it appends the test to `some-file.test.ts`, creating the file if necessary. We use `.spec.ts` as our test suffix, so there are extra steps involved to move the generated code into the spec file. I have prompted for Copilot to insert the test into a specific file: `create a test for xxx and insert the test into yyy.spec.ts file`, but it doesn't appear to understand that prompt.

**Comment by TonyDeLuca** [comment]
I am in the same boat. Our component being `some-file.ts` and `some-file.spec.ts` being the associated tests file.  It wants to create and place the tests in `some-file.test.ts` even when you ask it to create a new test from inside of the .spec.ts file. We cannot be the only people having this issue, unless we are the only people trying to use it add tests.
